.travis.yml

```diff
@@ -0,0 +1,134 @@
+---
+sudo: required
```

Comment:
@morvencao can you show some comments if we still need .tvais.yml here? I think we will use PROW